src/Styles/Navigation.css

```diff
@@ -4,39 +4,65 @@ nav {
   justify-content: space-between;
   align-items: center;
   margin: 0 auto;
-  height: 80px;
+  height: 94px;
   box-sizing: border-box;
-  padding: 0 30px;
-  margin-bottom: 30px;
+  padding: 24px 4%;
   background: white;
+  border-bottom: 1px solid #e8e8e8;
 }
 
 ul {
   display: flex;
   flex-direction: row;
   list-style: none;
-  gap: 20px;
+  gap: 40px;
 }
```

Comment:
- [ ] One of the project requirements is to follow [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and that means that If you want to create a CSS rule for multiple elements, use a class attribute instead of repeating the styling multiple times. I noticed you're using the `display:flex` styling multiple times in your CSS here so I'd advise you to create a separate class for example you can call it `.flex` and inside that class define the properties for a flexbox and use that class name wherever it's needed in your HTML elements. 
- [ ] I also noticed you're using the `color : #121212;` property multiple times in your CSS here so once again you can create a class for that style and use the class name wherever it's needed in your HTML instead.
- This comment applies to the rest of the style files as well so please revisit them and perform the necessary refactor.